### PR TITLE
Add simple mailer service to send user notifications

### PR DESCRIPTION
**Pull request by yellalena**
https://github.com/catbytes-community/webplatform-backend/issues/36

* Added `nodemailer` dependency and a `mailer_service` to send emails
* Updated configs
* Added triggering email sending on application status change
* Still waiting for real email templates from design and frontend, maybe will be replaced in a separate PR

### CODE REVIEW

File: services/mailer_service.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        <h2>Welcome to CatBytes!</h2>
+        <p>Hello, ${name}, we're happy to notify your application to CatBytes has been approved! :) </p>`;
+
+    return sendMail(email, "Welcome to CatBytes!", body);
+}
+
+async function sendApplicationRejectedEmail(email, name) {
+    // todo change template to real
+    const body = `
+        <h2>Uh-oh!</h2>
```
Comment (vm2052) [inline services/mailer_service.js:53]: love it

**Comment by vm2052** [comment]
honestly, no questions so far, amazing work 🥳

File: services/applications_service.js [hunk 2]
```diff
@@ -3,11 +3,13 @@ const repo = require('../repositories/applications_repository');
 async function getAllApplications() {
     return await repo.getAllApplications();
 }
+
 async function createNewApplication(name, about, email, video_link, discord_nickname) {
     return await repo.createNewApplication(name, about, email, video_link, discord_nickname);
 }
-async function updateApplicationStatus(id, status, comment, user_id) {
-    return await repo.updateApplicationStatus(id, status, comment, user_id);
+
```
Comment (marinacatbytes) [inline services/applications_service.js:10]: I think we need to update swagger here :)

File: services/mailer_service.js
Comment (marinacatbytes) [inline services/mailer_service.js:1]: perfection!

**Review by vm2052 — APPROVED** [review]

File: services/applications_service.js (hunk 2 above)
Comment (yellalena) [reply services/applications_service.js:10]: this is changed inside API and not recevied from client, so we're fine here. but thanks for pointing that out! 🙌🏻

**Pull request merged**